homeassistant/components/sensor/__init__.py

```diff
@@ -565,18 +565,19 @@ def state(self) -> Any:  # noqa: C901
                 )
             return value
 
-        # If the sensor has neither a device class, a state class nor
-        # a unit_of measurement then there are no further checks or conversions
+        prec = self.precision
```

Comment:
Any reason to not use a full name here?
```suggestion
        precision = self.precision
```